cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +636,76 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(2)
+@pytest.mark.parametrize(
+    "verbosity",
+    [
+        "1 (Verbose)",
+        "2 (More Verbose)",
+        "3 (Debug)",
+        "4 (Connection Debug)",
+        "5 (WinRM Debug)",
+        "0 (Normal)",
+    ],
+)
+@pytest.mark.meta(automates=[BZ(1460788)])
+def test_service_ansible_verbosity(
+    appliance,
+    request,
+    ansible_catalog_item,
+    ansible_service_catalog,
+    ansible_service_request,
+    ansible_service,
+    verbosity,
+):
+    """Check if the different Verbosity levels can be applied to service and
+    monitor the std out
+
+    Bugzilla:
+        1460788
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    pattern = '"verbosity"=>{}'.format(verbosity[0])
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {"verbosity": verbosity}
+        ansible_catalog_item.retirement = {"verbosity": verbosity}
+    # Log Validator
+    log = LogValidator(
+        "/var/www/miq/vmdb/log/evm.log", matched_patterns=[pattern]
+    )
+    # Start Log check or given pattern
+    log.fix_before_start()
+
+    @request.addfinalizer
+    def _revert():
+        with update(ansible_catalog_item):
+            ansible_catalog_item.provisioning = {"verbosity": verbosity}
+            ansible_catalog_item.retirement = {"verbosity": verbosity}
+
+        service = MyService(appliance, ansible_catalog_item.name)
+        if ansible_service_request.exists():
+            ansible_service_request.wait_for_request()
+            appliance.rest_api.collections.service_requests.action.delete(id=service_id.id)
+        if service.exists:
+            service.delete()
+
+    ansible_service_catalog.order()
+    ansible_service_request.wait_for_request()
+    request_descr = "Provisioning Service [{0}] from [{0}]".format(ansible_catalog_item.name)
+    service_id = appliance.rest_api.collections.service_requests.get(description=request_descr)
+    # Searching string '"verbosity"=>0' in evm.log as Standard Output is being
```

Comment:
Not searching 0 everytime here right? It should reflect what the user sets?

Reply:
It's just a comment to help to understand why I am using this parameter, nothing is being related to verbosity 0.

Reply:
It will gonna change as value of Verbosity is being changed.

Reply:
@sbulage I think just adding "Searching for "verbosity"=>0 (example) in evm.log as Standard Output is being logged to evm.log"